rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  belongs_to :student
+  belongs_to :book
+
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] Kindly make sure that the rental is adding itself to the @book achieving the 'belongs-to' association. Example :
```ruby    
book.rentals << self
```
Also, you have to do the same for the @person.